samigo/samigo-app/src/webapp/jsf/evaluation/totalScores.jsp

```diff
@@ -276,7 +276,7 @@ $(document).ready(function(){
        <h:outputText value="\"></a>" escape="false" />
 
        <h:commandLink styleClass="sam-scoretable-deleteattempt" title="#{commonMessages.delete_attempt}" action="totalScores" immediate="true" rendered="true" >
-         <h:outputText value="X" rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}" />
+         <h:outputText value="" styleClass="fa fa-trash" rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}" />
```

Comment:
Thanks for the PR.  As we're moving from text to an icon, we need to provide an alternative for screen readers. The easiest way is probably via an sr-only div with "Delete submission" as the text

Reply:
Good catch. I'll see if I can borrow (some of) the markup used for implementing the "Copy to clipboard" button now present in the published Settings view.